src/components/Integrations/Shopify/Shopify.js

```diff
@@ -61,35 +62,66 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
       </tbody>
     </table>
   );
+
+  const shopifyHeader = (
+    <>
+      <div className="block">
+        <h2>{_('shopify.header_title')}</h2>
+        <p>
+          <FormattedHTMLMessage id="shopify.header_subtitle" />
+        </p>
+      </div>
+      <hr />
+    </>
+  );
+
+  function useInterval(callback, delay) {
+    const savedCallback = useRef();
+
+    useEffect(() => {
+      savedCallback.current = callback;
+    }, [callback]);
+
+    useEffect(() => {
+      function tick() {
+        savedCallback.current();
+      }
+      if (delay !== null) {
+        let id = setInterval(tick, delay);
```

Comment:
Why did you use `let` in place of `const`? the `id` does not change, right?

Reply:
So it seems, I will change this...

Reply:
Maybe there is a reason, I only want to understand.

Reply:
Changed!